### ISSUE TITLE
Preview URL with File Upload in S3

### ISSUE THREAD

**Issue by MrMooky** [livewire/livewire#1091]
### Description
When uploading the preview to S3, the file is successfully uploaded into the temp folder, but the preview url is off.

```
'temporary_file_upload' => [
        'disk' => 's3'
]
```

```
AWS_BUCKET=cdn.domain.com
AWS_URL=https://cdn.domain.com
```

![screenshot](https://user-images.githubusercontent.com/3603346/84389210-0278d800-abf6-11ea-8881-f84e07622902.jpg)
**Context**
- Livewire version: 1.2.0
- Laravel version: 7.15.0
- Browser: Safari

**Comment by mblarsen** [comment]
How have you configured your bucket?

**Comment by MrMooky** [comment]
@mblarsen Yes, the bucket is working fine. The files are being uploaded successfully, the url ist just wrong. It appears though, that the URL starts with `https://s3.eu-central-1.amazonaws.com` by default. The actual bucket url is not taken into account here. But when I access data from that bucket, the defined url (AWS_URL) is used.

**Comment by merthurturk** [comment]
This issue is fixed with #1100 and merged, waiting to be tagged as a release

**Comment by techguydev** [comment]
what is the solution for this @merthurturk =
